test/test_helper.rb

```diff
@@ -5,6 +5,9 @@
   SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 end
 
+require "codeclimate-test-reporter"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)